src/main/kotlin/dev/supergrecko/kllvm/core/typedefs/Builder.kt

```diff
@@ -67,12 +67,12 @@ public class Builder internal constructor(internal val llvmBuilder: LLVMBuilderR
     override fun close() = dispose()
 
     companion object {
-        //region Core::Builders
+        //region InstructionBuilders
         @JvmStatic
-        fun create(ctx: Context? = null): Builder {
+        fun create(ctx: Context? = Context.getGlobalContext()): Builder {
             return Builder(LLVM.LLVMCreateBuilderInContext(ctx?.llvmCtx))
```

Comment:
Could remove the ?'s from ctx and Context

Reply:
Should be fixed now